src/main/resources/templates/partials/header.hbs

```diff
@@ -6,7 +6,7 @@
         </li>
 
         <li>
-                     <form action="search" method="GET" id="headerform">
+                     <form action="/search" method="GET" id="headerform">
```

Comment:
Please add to summary and menition the issue

Reply:
have